### PR TITLE
Move the nominatim utils in a dedicated class.

### PR DESCRIPTION
**Pull request by SPalominos**
Move the nominatim utils in a dedicated class.

### CODE REVIEW

File: osm/src/main/groovy/org/orbisgis/orbisanalysis/osm/utils/NominatimUtils.groovy [hunk 1]
```diff
@@ -0,0 +1,169 @@
+package org.orbisgis.orbisanalysis.osm.utils
```
Comment (ebocher) [inline osm/src/main/groovy/org/orbisgis/orbisanalysis/osm/utils/NominatimUtils.groovy:37]: Add header and authors

File: osm/src/test/groovy/org/orbisgis/orbisanalysis/osm/AbstractOSMTest.groovy [hunk 2]
```diff
@@ -106,8 +107,8 @@ abstract class AbstractOSMTest {
     void afterEach(){
         //Restore the modified object
         Utilities.metaClass.static.executeOverPassQuery = executeOverPassQuery
-        Utilities.metaClass.static.getAreaFromPlace = getAreaFromPlace
-        Utilities.metaClass.static.executeNominatimQuery = executeNominatimQuery
+        NominatimUtils.metaClass.static.getArea = getArea
```
Comment (ebocher) [inline osm/src/test/groovy/org/orbisgis/orbisanalysis/osm/AbstractOSMTest.groovy]: I'd prefer the name getAreaFromPlace

Comment (ebocher) [reply osm/src/test/groovy/org/orbisgis/orbisanalysis/osm/AbstractOSMTest.groovy]: or getPlace

File: osm/src/main/groovy/org/orbisgis/orbisanalysis/osm/utils/NominatimUtils.groovy (hunk 1 above)
Comment (SPalominos) [reply osm/src/main/groovy/org/orbisgis/orbisanalysis/osm/utils/NominatimUtils.groovy:37]: header added, authors already present (see line 17-19)

File: osm/src/test/groovy/org/orbisgis/orbisanalysis/osm/AbstractOSMTest.groovy (hunk 2 above)
Comment (SPalominos) [reply osm/src/test/groovy/org/orbisgis/orbisanalysis/osm/AbstractOSMTest.groovy]: Name changed for 'getPlace' (shorter)

**Pull request closed without merging**